theia-deb-build-docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:bionic
+# install development tools
+RUN apt-get update && \
+    apt-get install -y curl sudo build-essential jq vim && \
+    curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash - && \
+    apt-get install -y nodejs && \
+    npm install -g yarn
+
+# add theia installation file
+COPY package.json run_theia.sh /theia-app/
+RUN chmod +x /theia-app/run_theia.sh
+
+# build theia
+WORKDIR /theia-app
+RUN yarn && \
+    NODE_OPTIONS="--max_old_space_size=4096" yarn theia build && \
+    yarn autoclean --init && \
+    echo *.ts >> .yarnclean && \
+    echo *.ts.map >> .yarnclean && \
```

Comment:
I guess it would be an option to package the `.ts` and `.ts.map` files separately in their own .deb package, instead of deleting them? i.e. a bit like for some C apps, one can sometimes install the `<app>-debug` package, if the debug symbols are needed.
(does not need to be done as part of this PR necessarily)

Reply:
Certainly. I only remove those files because the other examples in the repo do it. I wanted to produce the smallest deb package possible. I don't know if the [node-deb](https://www.npmjs.com/package/node-deb) module supports creating a debug package explicitly. It has good docs and lots of examples but makes no reference to building debug packages.

I suppose one could leave the source files alone (not remove them) and then continue the build process in order to create an `<app>-debug` package.